bindings/python/htcondor2/_daemon_type.py

```diff
@@ -1 +1,60 @@
+import sys
 
+if sys.version_info < (3, 5, 0):
+    import compat_enum as enum
+else:
+    import enum
+
+
+class DaemonType(enum.IntEnum):
+    """
+    An enumeration of known daemon types.
+
+    .. attribute:: none
+
+        Any type of daemon; useful when specifying queries where all matching
+        daemons should be returned.
+
+    .. attribute:: Any
+
+        Any type of daemon; useful when specifying queries where all matching
+        daemons should be returned.
+
```

Comment:
Sub-description for any is duplicated under the none option

Reply:
The version 1 documentation to be confusing (it wasn't clear if `None` was undocumented or a duplicate of `Any`), so I made it an explicit duplicate.

(Interestingly, capital-N `None` -- as specified in the version 1 documentation -- is, as far as I can tell, always a syntax error.  (`None` always means the Python `None`, and can't ever refer to anything else, like an object member.))

Having looked at the code a bit more closely, it's clear that `DT_NONE` is not treated the same as `DT_ANY`.  For now, given that it was unusable in version 1, I've just undocumented it in version 2, and added some uninformative documentation for the other enumeration values.